x/nft/username/handler.go

```diff
@@ -249,3 +249,71 @@ func findActor(h tokenHandler, ctx weave.Context, t Token) weave.Address {
 	}
 	return nil
 }
+
+type AddRemoveApprovalHandler struct {
+	tokenHandler
+}
+
+func (h *AddRemoveApprovalHandler) Check(ctx weave.Context, store weave.KVStore, tx weave.Tx) (weave.CheckResult, error) {
+	var res weave.CheckResult
+	if _, err := h.validate(ctx, tx); err != nil {
+		return res, err
+	}
+	res.GasAllocated += createUsernameCost
```

Comment:
req: please use a const based on the message type. I don't know if we would have separate costs for add and remove operations.

Reply:
@alpe this was just an example of implementation.
It's gonna change with the new approach